.github/workflows/check-pr-max-lines.yml

```
@@ -1,16 +1,19 @@
-name: "Check PR Max Lines"
+name: 'Check PR Max Lines'
 
 on:
   pull_request:
     types: [opened, reopened, synchronize]
 
 permissions:
   contents: read
-  pull-requests: write  # Required for the called workflow to manage labels
+  pull-requests: write # Required for the called workflow to manage labels
 
 jobs:
   check-pr-max-lines:
-    uses: metamask/github-tools/.github/workflows/pr-line-check.yml@main
-    with:
-      max_lines: 1000
-      ignore_patterns: '(\.lock|\.snap)$'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR lines changed
+        uses: MetaMask/github-tools/.github/actions/pr-line-check@072c900e2447470ad7ad5d8a74c14f253d583d9f
```

Once [the release of `github-tools`](https://github.com/MetaMask/github-tools/pull/174) is merged, we should use `v1`.


```suggestion
        uses: MetaMask/github-tools/.github/actions/pr-line-check@v1
```